homeworks/kostiantyn.nosatskyi_NosatskyiK/PokemonProject/style.css

```diff
@@ -0,0 +1,28 @@
+h1 {
+  font-size: 48px ;
+  font-family: Roboto, serif ;
+  font-weight: 700 ;
+}
+
+.main_text {
+  font-family: Roboto, serif ;
+  font-size: 22px ;
+  font-weight: 400 ;
+}
+
+/* додав розміри для article щоб відповідати макету */
+
+article {
+  width: 575px ;
```

Comment:
Better not to fix both `width` and `height` of blocks, which content may be changed. It could lead to a broken layout
<img width="1637" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/0f823c23-de55-4eef-8da2-be2706e18a7c">